tests/chainer_tests/distributions_tests/test_multinomial.py

```diff
@@ -0,0 +1,68 @@
+import numpy
+
+from chainer import cuda
+from chainer import distributions
+from chainer import testing
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(2, 3), ()],
+    'is_variable': [True, False],
+    'sample_shape': [(3, 2), ()],
+    'k': [3],
+}))
+@testing.fix_random()
+@testing.with_requires('scipy')
```

Comment:
@YoshikawaMasashi Would you apply the following change?
```suggestion
@testing.with_requires('scipy>=0.19')
```